networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/ActiveEth2Network.java

```diff
@@ -107,7 +106,6 @@ public ActiveEth2Network(
     this.peerManager = peerManager;
     this.eventBus = eventBus;
     this.recentChainData = recentChainData;
-    this.gossipEncoding = gossipEncoding;
```

Comment:
I think its probably better to continue just using the gossipEncoding passed in here - especially since it's still being passed into the constructor anyway.

Reply:
Yep, totally makes sense with the final PR version (this was done for the previous PR incarnation)